waspc/src/Wasp/AI/GenerateNewProject/Plan.hs

```diff
@@ -99,6 +99,8 @@ generatePlan newProjectDetails planRules = do
         Typically, plan will have AT LEAST one query, at least one action, at least one page, and at
         least two entities. It will very likely have more than one of each, though.
 
+        DO NOT create actions for login and logout under any circumstances. They are already included in Wasp.
```

Comment:
If the user's prompt mentions anything about logging in and logging out, it sometimes creates actions for those functionalities. Hopefully this helps.

Reply:
What we can also potentially do to handle this is to add a bit of logic in Plan.hs, under `fixPlan`, that checks if any action is named `login` or `logout` or maybe contains those words -> and if there are such actions, we warn it there that we suspect those to be redundant because it is already included in wasp and that it should remove them.